### ISSUE TITLE
extend format de15 mapping

### ISSUE THREAD

**Issue by gertho** [miku/span#4]
Would you mind to extend the format mapping table under https://github.com/miku/span/blob/master/assets/finc/formats/de15.json to the entire format_map_de15.properties, we might need to map more than the existing formats for [BASE](https://projekte.ub.uni-leipzig.de/issues/5994). 

# format_map_de15.properties
> \#
> \# DATA FROM INDEXER = Label
> \#
> Book                               = Book, E-Book
> eBook                              = Book, E-Book
> ElectronicBook                     = Book, E-Book
> ElectronicExhibitionCatalog        = Book, E-Book
> ElectronicSpecialPrint             = Book, E-Book
> ElectronicFestschrift              = Book, E-Book
> ElectronicJournal                  = Journal, E-Journal
> ElectronicSerial                   = Journal, E-Journal
> Journal                            = Journal, E-Journal
> Serial                             = Journal, E-Journal
> ElectronicNewspaper                = Journal, E-Journal
> ElectronicSchoolPublication        = Journal, E-Journal
> Newspaper                          = Journal, E-Journal
> Article                            = Article, E-Article
> ElectronicArticle                  = Article, E-Article
> ElectronicThesis                   = Thesis
> Thesis                             = Thesis
> Atlas                              = Map
> Globe                              = Map
> Map                                = Map
> ElectronicMap                      = Map
> AudioTape                          = Audio
> CD                                 = Audio
> DVDAudio                           = Audio
> MusicRecording                     = Audio
> ElectronicMusicRecording		   = Audio
> Record                             = Audio
> SoundRecordingMedium               = Audio
> SoundCassette                      = Audio
> SoundDisc                          = Audio
> SoundRecording                     = Audio
> NaxosCD                            = Audio
> AudioVisualMedia                   = Video
> BluRayDisc                         = Video
> DVDVideo                           = Video
> Filmstrip                          = Video
> MotionPicture                      = Video
> Video                              = Video
> VideoTape                          = Video
> VideoCartridge                     = Video
> VideoCassette                      = Video
> VideoDisc                          = Video
> VideoReel                          = Video
> VideoRemote                        = Video
> ArtPrint                           = Visual Media
> Chart                              = Visual Media
> Collage                            = Visual Media
> Drawing                            = Visual Media
> FlashCard                          = Visual Media
> Painting                           = Visual Media
> Photo                              = Visual Media
> Photonegative                      = Visual Media
> Placard                            = Visual Media
> Print                              = Visual Media
> SensorImage                        = Visual Media
> Slide                              = Visual Media
> Transparency                       = Visual Media
> ElectronicVisualMedia              = Visual Media
> CDROM                              = Software
> ChipCartridge                      = Software
> DiscCartridge                      = Software
> DVDROM                             = Software
> FloppyDisk                         = Software
> Software                           = Software
> TapeCartridge                      = Software
> TapeCassette                       = Software
> TapeReel                           = Software
> Microfiche                         = Microform
> Microfilm                          = Microform
> Microform                          = Microform
> MusicalScore                       = Musical Score
> NotatedMusic                       = Musical Score
> ElectronicMusicalScore             = Musical Score
> Braille                            = Braille
> Kit                                = Kit
> Manuscript                         = Manuscript
> ElectronicManuscript               = Manuscript
> Nachlass                           = Nachlass
> PhysicalObject                     = Physical Object
> Unknown                            = Unknown Format
> Electronic                         = Website
> ElectronicIntegratingResource      = Website
> ElectronicResourceDataCarrier      = Electronic Resource (Data Carrier)
> ElectronicResourceRemoteAccess     = Electronic Resource (Remote Access)
>  
> \# 980b source_id
> \# 54 = Book, E-Book # see format.bsh erSet
> 
> \# Ticket #4911
> 52 = Book, E-Book
> 62 = Book, E-Book
> 73 = Article, E-Article
> 78 = Electronic Resource

**Comment by miku** [comment]
Thanks for the input. 

* [prop2js](https://github.com/nzakas/props2js)

```shell
$ java -jar props2js-0.1.0.jar import/translation_maps/format_map_de15.properties -t json
{
  "Painting": "Visual Media",
  "VideoCartridge": "Video",
  "Software": "Software",
  "Electronic": "Website",
  "Atlas": "Map",
  ...
}
```

* [refs 8844#note-10]

**Comment by miku** [comment]
The updated mapping should be available in [v0.1.189](https://github.com/miku/span/releases/tag/v0.1.189) or later.
